dashboard/test/models/blockly_test.rb

```diff
@@ -209,6 +209,55 @@ def assert_equal_xml(a, b)
     assert_equal localized_custom_block, translated_block
   end
 
+  test 'return default value if string is not translated' do
+    test_locale = :"te-ST"
+    I18n.locale = test_locale
+    custom_i18n = {
+      "data" => {
+        "blocks" => {
+          "DanceLab_jumpTo" => {
+            "text" => "springen naar {TIMESTAMP} {DIRECTION}",
+            "options" => {
+              "DIRECTION" => {
```

Comment:
I'd also test this against the scenario in which the _keys_ for the translation don't exist